test/flycheck-test.el

```diff
@@ -3053,6 +3053,13 @@ See https://github.com/flycheck/flycheck/issues/531 and Emacs bug #19206"))
    "language/erlang/rebar3/src/erlang-warning.erl" 'erlang-mode
    '(6 nil warning "wrong number of arguments in format call" :checker erlang-rebar3)))
 
+(flycheck-ert-def-checker-test erlang-rebar3 erlang build
+  (flycheck-ert-should-syntax-check
+   "language/erlang/rebar3/_checkouts/dependency/src/dependency.erl" 'erlang-mode)
+  (should (not (file-exists-p
+                (flycheck-ert-resource-filename
+                 "language/erlang/rebar3/_build/default/lib/dependency/_build")))))
```

Comment:
Based on the comment above about the regex, this assertion is testing that when we open a buffer for a dependency, it doesn't build the dependency as if it were a standalone project. If it did, it would create that `_build` directory, so we're asserting that it doesn't.